docs/guides/exchange_guide.md

```diff
@@ -84,13 +87,19 @@ Once you have everything needed to use the `wallet.rs` library, it is necessary
 
 The storage is encrypted at rest, so you need a strong password and location where to put your storage. 
 
-*Note: manage your password with the utmost care*.
+:::Note
+Manage your password with the utmost care.
+
+:::
 
 Technically speaking, "storage" means a single file called `wallet.stronghold`. It is also needed to generate a seed (mnemonic) that serves as a cryptographic key from which all accounts and related addresses are generated.
 
 One of the key principles behind the `stronghold`-based storage is that no one can get a seed from the storage. You deal with all the accounts purely via the `Account_Manager` instance where all complexities are hidden under the hood and are dealt with in a secure way. In case you would also like to store a seed somewhere else, there is another method, `AccountManager.generateMnemonic()`, that generates random seeds. This method can be leveraged before the actual account initialization.
```

Comment:
```suggestion
One of the key principles behind the `stronghold`-based storage is that no one can get a seed from the storage. You deal with all the accounts purely via the `Account_Manager` instance where all complexities are hidden under the hood and are dealt with securely. In case you would also like to store a seed somewhere else, there is another method, `AccountManager.generateMnemonic()`, that generates random seeds. This method can be leveraged before the actual account initialization.
```